MetaMask/metamask-mobile
# feat: hide alert when navigating to protect wallet

**Description**

This PR aims to solve a bug on the backup alert, when navigating to the protect your wallet screen the alert kept being visible.

This PR contains BackupAlert converted to typescript and functional component.

## **Manual testing steps**

_1. Step1: Create a new wallet
_2. Step2: Do not protect your wallet pressing (remember later)
_3. Step3: Press protect your wallet on the backup alert
_4. Step4: The protect your wallet should not be there

## **Screenshots/Recordings**
https://recordit.co/skV2sxdzil

new Bell Icon:
![image](https://github.com/MetaMask/metamask-mobile/assets/46944231/71b95e66-8d71-4951-a02e-4beca3e07af5)
![image](https://github.com/MetaMask/metamask-mobile/assets/46944231/7a9afc84-4138-4d91-9f8e-4f564b757125)

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #4400 

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
